recipes/matlab-runtime-9.7/recipe.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: matlab-runtime-9.7
+  version: 9
+
+source:
+  url: https://ssd.mathworks.com/supportfiles/downloads/R2019b/Release/9/deployment_files/installer/complete/glnxa64/MATLAB_Runtime_R2019b_Update_9_glnxa64.zip
+  sha256: d213e2960a20332f121ec4addf81469b2a2aab2ad8b4ed4226875da6e8af7c65
+
+build:
+  number: 0
+  script:
+    - ./install -mode silent -agreeToLicense yes -outputFile install_log.txt -destinationFolder $PREFIX/MATLAB/MATLAB_Runtime
```

Comment:
Two minor remarks that are more a personal feeling than technical points.
- I would use lower case for directories since, once package is installed, all other sibling directories are lower case so it makes this MATLAB directory special.
- If it is very unlikely that something else than Matlab runtime is packaged, the MATLAB directory level could be removed.

Reply:
I agree with you on principle, but I used to the defaults of the MathWorks installer.